examples/README.md

```diff
@@ -11,25 +11,25 @@ md_toc github examples/README.md | sed -e "s/\s-\s/ * /"
 * [Instructions](#instructions)
   * [(Optional) Setting up a remote terraform state](#optional-setting-up-a-remote-terraform-state)
 * [Blueprint Descriptions](#blueprint-descriptions)
-  * [hpc-cluster-small.yaml](#hpc-cluster-smallyaml-)
-  * [hpc-cluster-high-io.yaml](#hpc-cluster-high-ioyaml-)
-  * [slurm-gcp-v5-hpc-centos7.yaml](#slurm-gcp-v5-hpc-centos7yaml-)
-  * [slurm-gcp-v5-ubuntu2004.yaml](#slurm-gcp-v5-ubuntu2004yaml-)
-  * [slurm-gcp-v5-high-io.yaml](#slurm-gcp-v5-high-ioyaml-)
-  * [image-builder.yaml](#image-builderyaml-)
-  * [hpc-cluster-intel-select.yaml](#hpc-cluster-intel-selectyaml-)
-  * [daos-cluster.yaml](#daos-clusteryaml-)
-  * [daos-slurm.yaml](#daos-slurmyaml-)
-  * [hpc-cluster-amd-slurmv5.yaml](#hpc-cluster-amd-slurmv5yaml-)
-  * [cloud-batch.yaml](#cloud-batchyaml--)
-  * [batch-mpi.yaml](#batch-mpiyaml-)
-  * [spack-gromacs.yaml](#spack-gromacsyaml--)
-  * [omnia-cluster.yaml](#omnia-clusteryaml--)
-  * [hpc-cluster-small-sharedvpc.yaml](#hpc-cluster-small-sharedvpcyaml--)
-  * [hpc-cluster-localssd.yaml](#hpc-cluster-localssdyaml--)
-  * [htcondor-pool.yaml](#htcondor-poolyaml--)
-  * [quantum-circuit-simulator.yaml](#quantum-circuit-simulatoryaml-)
-  * [starccm-tutorial.yaml](#starccm-tutorialyaml--)
+  * [hpc-cluster-small.yaml](#hpc-cluster-smallyaml-) ![core-badge]
```

Comment:
In our modules, we broke down the modules by categories, and then, within every category, we sorted them showing first core, then community, then community experimental. I am ok with submitting this as is, since it is already an improvement over the current state, but we could also consider resorting these examples.

Reply:
I'll add a task for that. Rather than do it myself, I'd rather wait and have a discussion about the best way to organize the blueprints.